.travis.yml

```diff
@@ -24,10 +24,11 @@ script:
  - sleep 30
  - ./adop workspace -w Travis create
  - ./adop project -p TestProject -w Travis create
+ - ./adop project -p TestProject -w Travis load_collection -c https://raw.githubusercontent.com/Accenture/adop-docker-compose/master/test/resources/cartridge_collection.json
```

Comment:
@anton-kasperovich I think I'll need to split this PR into two, unless I reference another cartridge collection (e.g. DOA) but I don't like this idea as the test should be decoupled. Therefore I feel I should break this PR one for:

- adding the test cartridge collection JSON
- Updating travis file to test collection